### PR TITLE
Use uimodel for staffs viewmodel

### LINKED ISSUE #360: Use UiModel for StaffsViewModel
## Kind (Required)

- Improvement

 ## Overview (Required)

- At first, we did not use UiModel.
- I want to use UiModel for it 🙏 
You can access staffs screen from about screen

### PR DESCRIPTION
**Pull request by ho2ri2s**
## Issue
- close #360 

## Overview (Required)
- use UiModel for StaffsViewModel
  - observe it at StaffsFragment

## Links
-

## Screenshot
committee_git

### CODE REVIEW

**Comment by ho2ri2s** [comment]
Can I challenge #361 ?

**Comment by yurihondo** [comment]
👀

**Comment by yurihondo** [comment]
Thank you for your helpful cooperation.

> Can I challenge #361 ?
Yes of course!
please support it 🙏 I'll assigned you later 🙇

**Comment by e10dokup** [comment]
@ho2ri2s Could you please write comment to issue #360? We need you to participate in this issue for us to assign 🙇

File: feature/staff/src/main/java/io/github/droidkaigi/confsched2020/staff/ui/StaffsFragment.kt [hunk 1]
```diff
@@ -73,26 +74,21 @@ class StaffsFragment : Fragment() {
...
-                    state.e.printStackTrace()
-                }
-            }
+        staffsViewModel.uiModel.observe(viewLifecycleOwner) { uiModel ->
+            progressTimeLatch.loading = uiModel.isLoading
+            groupAdapter.update(uiModel.staffContents.staffs.map {
+                staffItemFactory.create(it)
+            })
+
+            uiModel.error?.printStackTrace()
```
Comment (yurihondo) [inline feature/staff/src/main/java/io/github/droidkaigi/confsched2020/staff/ui/StaffsFragment.kt]: Could you please pass an error to system like so?
https://github.com/DroidKaigi/conference-app-2020/blob/master/feature/announcement/src/main/java/io/github/droidkaigi/confsched2020/announcement/ui/AnnouncementFragment.kt#L79

**Comment by ho2ri2s** [comment]
@yurihondo 
> please support it 🙏 I'll assigned you later 🙇

thanks!

@e10dokup 
> Could you please write comment to issue #361?

sorry, I did it 🙇‍♂️

File: feature/staff/src/main/java/io/github/droidkaigi/confsched2020/staff/ui/StaffsFragment.kt (hunk 1 above)
Comment (ho2ri2s) [reply feature/staff/src/main/java/io/github/droidkaigi/confsched2020/staff/ui/StaffsFragment.kt]: Wrongly, I thought dynamic feature module cannot depend on system module...
I fix it at 67ea11d .
Please review🙇‍♂️

**Pull request merged**